test/benchmarks/driverBench/index.js

```diff
@@ -41,7 +41,7 @@ function makeBSONLoader(fileName) {
     const EJSON = require('mongodb-extjson');
     EJSON.setBSONModule(BSON);
 
-    this.bson = new BSON();
+    this.bson = BSON;
```

Comment:
To refer to an earlier comment: `this.bson` should only be used in cases where we want to optionally accept a user-provided `BSON` instance (perhaps for testing with their cool-new-bson-library). In all other cases, now that `BSON` is just a namespace, we should use it directly from `retrieveBSON` or perhaps in these benchmarks just by `require('BSON')`